### PR TITLE
Added and configured linters

### LINKED ISSUE #69: Configure EsLint At The Project To Fix Code Style
### Explanation:

EsLint is not working on the project, this issue is about to check why it's not working and fixing it. 

### Checklist
 This issue can be closed when the following tasks are completed:
 
 - [x] EsLint is in the project.
 - [x] EsLint have a script command to execute it.
 - [ ] - [x] EsLint is detecting warnings and errors and fixing it.

### PR DESCRIPTION
**Pull request by e-azocar**
### Description

I installed and configures the linters `eslint` to Javascript code and `stylelint` to CSS code, I added the command `npm run lint` or `yarn lint` to verify the correct syntax

Also, I installed `prettier` to have a code correctly formatted, you can test it with `npm run format` or `yarn format`

fixes #69 

#### Screenshots
image

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+	"env": {
+		"browser": true,
+		"es2021": true
```
Comment (Israel-Laguan) [inline .eslintrc.json]: JSON files seem to have 4 spaces indentation? JS files looks to have 2

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -1,9 +1,8 @@
 import React from "react";
 import LogoFooter from "../assets/img/logo.png";
-import notLoggedButtons from './isNotLogged';
+import notLoggedButtons from "./isNotLogged";
```
Comment (heliomar-pena) [inline src/components/Footer.jsx:3]: Can you add a rule to EsLint to change double quotes for simple quotes in JavaScript code?

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
Please check comments

**Comment by heliomar-pena** [comment]
Please rebase to get the last changes

**Pull request closed without merging**